requirements.txt

```diff
@@ -1,102 +1,108 @@
 #
-# This file is autogenerated by pip-compile with Python 3.12
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
Also, this bot is running python 3.11

I believe that this was switched to 3.12 by mistake. (it's just a comment so it doesn't really matter)